src/main/java/Duke.java

```diff
@@ -6,5 +11,44 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        Scanner in = new Scanner(System.in);
+        String input;
+        ArrayList<String> items = new ArrayList<String>();
+        ArrayList<Boolean> marked = new ArrayList<Boolean>();
```

Comment:
Using two resizable arrays may affect runtime and make it harder to code as there needs to be constant linkage between the two items in each array. Although it may seem simple now, this will become much harder to maintain when you have more methods being implemented into your code in the future, such as deleting and moving items along the list. Hence, I believe that creating a separate class Task may benefit the code and yourself greatly.